clearcut_detection_backend/frontend/src/App.css

```diff
@@ -1,5 +1,8 @@
 .App {
-  background: #282c34;
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  background: #202020;
```

Comment:
if you need to set this styles to sidebar, please, move them to sidebar component